Tasks/AzureWebPowerShellDeployment/task.json

```diff
@@ -0,0 +1,122 @@
+{
+    "id": "DCBEF2C9-E4F4-4929-82B2-EA7FC9166109",
+    "name": "AzureWebPowerShellDeployment",
+    "friendlyName": "Azure App Service: Classic (Deprecated)",
+    "description": "Create or update Azure App Service using Azure PowerShell",
+    "helpMarkDown": "More Information : “Azure App Service: Classic” Task will be deprecated. We recommend migrating your definition to use “Azure App Service Deploy” Task instead. For more information please refer [this](https://go.microsoft.com/fwlink/?LinkID=613750).",
+    "category": "Deploy",
+    "visibility": [
+        "Build",
+        "Release"
+    ],
+    "runsOn": [
+        "Agent"
+    ],
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 46
```

Comment:
I assume the patch version was bumped up.

Reply:
yes